prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1572,7 +1508,7 @@ int rxr_ep_grow_rx_pkt_pools(struct rxr_ep *ep)
 	int err;
 
 	assert(ep->efa_rx_pkt_pool);
-	err = ofi_bufpool_grow(ep->efa_rx_pkt_pool);
+	err = ofi_bufpool_grow(ep->efa_rx_pkt_pool->localinfo_pool);
```

Comment:
You need to also grow `wiredata_pool` here.

I think it is better to have a function `rxr_pkt_pool_grow` .to grow the packet pool together can be called here.

Reply:
added new method `rxr_pkt_pool_grow`